engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -595,7 +595,7 @@ public void doInTransactionWithoutResult(final TransactionStatus status) {
                 if (_networkOfferingDao.findByUniqueName(NetworkOffering.DefaultIsolatedNetworkOfferingForVpcNetworks) == null) {
                     offering = _configMgr.createNetworkOffering(NetworkOffering.DefaultIsolatedNetworkOfferingForVpcNetworks,
                             "Offering for Isolated VPC networks with Source Nat service enabled", TrafficType.Guest, null, false, Availability.Optional, null,
-                            defaultVPCOffProviders, true, Network.GuestType.Isolated, false, null, false, null, false, false, null, false, null, true, true, false, false, null, null, null,true, null, null, false);
+                            defaultVPCOffProviders, true, Network.GuestType.Isolated, false, null, true, null, false, false, null, false, null, true, true, false, false, null, null, null,true, null, null, false);
```

Comment:
so only this needs backporting to 4.19?

Reply:
this part, and also the SQL to 4.19.2->4.19.3 upgrade schema

Reply:
checked with Pearl, currently therre is no upgrade path for 4.19.2 to 4.19.3

old envs deployed before 4.19.0 will be good when they upgrade to 4.19.0 or later
the impacted versions are old envs deployed with 4.19.0/4.19.1/4.19.2
they will be good when upgrade to 4.20.1
but they will have the issue when upgade to 4.19.3, if we do not add upgrade path with db change
the db change is quite simple ..

we can backport the java change to 4.19.3, so 4.19.3 fresh env will be fine.

Reply:
ok, I have a boit of an issue with backporting the sql change, not of principle but because there is no other reason to create an upgrade path yet. I think this issue is too low priority to create the upgrade path for the sake of it. what do you think @Pearl1594 @weizhouapache ?

Reply:
I agree @DaanHoogland 

maybe you can add SQL changes to the description of the PR #10843 
in case someone want the fix in 4.19.1/2/3, they can execute the SQL